post: go-live (#7)
rero/sonar.ch: content/post/sonar-goes-live.md

---
title: "SONAR goes live"
date: 2021-05-05
draft: false
tags: ["project", "shared portal", "dedicated portal", "open access"]
---

After the successful conclusion of the SONAR development project, the service is ready for its go-live and does so with its very first client : [Haute Ecole Pédagogique du Valais](http://www.hepvs.ch/). As RERO+ has been officially established as a private foundation, this is an exciting time to get a glimpse into the future of SONAR.

<!--more-->

## A federating project

> Part of the P-5 program (Scientific information: access, processing and safeguarding) of swissuniversities, SONAR was developed in order to provide a national platform for Swiss open access scientific publications. Information technologists and information scientists from all language areas of Switzerland collaborated for two years to carry out the project : [RERO](https://www.rero.ch/), [HES-SO](https://www.hes-so.ch/) / [HEG Genève](https://www.hesge.ch/heg/), [FH Graubünden](https://www.fhgr.ch/), [Università della Svizzera italiana](https://www.usi.ch/).

Reviewer: ```suggestion
Part of the P-5 program (Scientific information: access, processing and safeguarding) of [swissuniversities](https://www.swissuniversities.ch/), SONAR was developed in order to provide a national platform for Swiss open access scientific publications. Four partners from all language areas of Switzerland collaborated during two years to carry out the project : [RERO](https://www.rero.ch/), [HES-SO](https://www.hes-so.ch/) / [HEG Genève](https://www.hesge.ch/heg/), [FH Graubünden](https://www.fhgr.ch/), [Università della Svizzera italiana](https://www.usi.ch/).
```

> Now that the project is live, RERO+ will take over the continuous development of SONAR and act as service provider. 

Reviewer: To remove. I added this point at the end.

```suggestion
```

## SONAR's live services

> 

Reviewer: Here I would put emphasis on the 2 services (global and dedicated/HEP-VS), either by using a bullet list or bold characters. Out of this list, we keep the mention of the shared portal, but it has not yet clients in production.

> SONAR provides different services with firstly a [global portal](https://sonar.ch/) that aggregates all Swiss open access publications through content tracking. This mechanism harvests data from national and international repositories and creates a database. SONAR global hosts these publications' metadata but not the actual documents. These are nonetheless available through links directing to the source servers. It does not require any action from editors and external open access portals.

Reviewer: ```suggestion
SONAR provides different services with firstly a [global portal](https://sonar.ch/) that aggregates all Swiss open access publications through content tracking. This mechanism harvests metadata from national and international repositories and creates a database. For each document with a full-text, a link to the source server (in principle the repository or the editor website) is available. It does not require any action from editors and external open access portals.
```

I don't really understand the part "It does not require any action from editors and external open access portals." and would remove it.

> For institutions that need it, SONAR can also act as a dedicated portal to host any type of digital library, not necessarliy open access. This type of portal benefits from an extended customer support and can be extensively customized by the client with its own visual identity as well as specific metadata fields. The [Haute Ecole Pédagogique du Valais](lien vers le portail dédié) uses this service and is SONAR's first client to go live.

Reviewer: ```suggestion
For institutions that need it, SONAR can also act as a dedicated portal to host any type of digital library, not necessarliy open access. This type of portal benefits from an extended customer support and can be extensively customized by the client with its own visual identity as well as specific metadata fields. The [Haute école pédagogique du Valais](lien vers le portail dédié) uses this service and is SONAR's first client to go live.
```

*capture d'écran du portail HEPVS*

A third service (the shared portal) is also available for institutions that need to host their open access publications. Less expensive and less customizable, this service offers a visibility of scholarly publications through the global portal and creates a dedicated collection for the client institution.

## The future

> Two other institutions confirmed their interest in their own dedicated portals and migration projects are in progress. These institutions are the [USI](https://www.usi.ch/) and [HEP-BEJUNE](https://www.hep-bejune.ch/).

Reviewer: Je reformulerais ainsi:
Two other institutions chose SONAR as the solution for their institutional archive, also preferring the dedicated option:

* [Università della Svizzera italiana](https://www.usi.ch/)
* [HEP-BEJUNE](https://www.hep-bejune.ch/).

In parallel to these two migration projects, RERO+ continues to ensure the ongoing development of the solution.

---

> title: "SONAR goes live"

Reviewer: An idea:

```suggestion
title: "Go live! Goodbye SONAR project, hello SONAR services"
```

date: 2021-05-05
draft: false
tags: ["project", "shared portal", "dedicated portal", "open access"]
---

> After the successful conclusion of the SONAR development project, the service is ready for its go-live and does so with its very first client : [Haute école pédagogique du Valais](http://www.hepvs.ch/). As RERO+ has been officially established as a private foundation, this is an exciting time to get a glimpse into the future of SONAR.

Reviewer: J'avais déjà rédigé cette phrase. A récupérer éventuellement pour l'intro?

> After two years of intense work, cross-institutional exchanges, lively debates, feasibility analysis and exploratory tests, the SONAR project ended in December 2020.

Author: C'est déjà la première phrase du billet précédent. Ca ne me paraît pas pertinent de le répéter tel quel.

Reviewer: Ok, je me suis mélangé entre les billets de blog. Pareil pour le commentaire précédent.

<!--more-->

> ## A federating project

Reviewer: ```suggestion
## From the project...
```

> Part of the P-5 program (Scientific information: access, processing and safeguarding) of [swissuniversities](https://www.swissuniversities.ch/), SONAR was developed in order to provide a national platform for Swiss open access scientific publications. Four partners from all language areas of Switzerland collaborated during two years to carry out the project : [RERO](https://www.rero.ch/), [HES-SO](https://www.hes-so.ch/) / [HEG Genève](https://www.hesge.ch/heg/), [FH Graubünden](https://www.fhgr.ch/), [Università della Svizzera italiana](https://www.usi.ch/).

Reviewer 2: ```suggestion
Part of the P-5 program (Scientific information: access, processing and safeguarding) of [swissuniversities](https://www.swissuniversities.ch/), SONAR was developed in order to provide a national platform for Swiss open access scientific publications. Four partners from all language areas of Switzerland collaborated during two years to carry out the project: [RERO](https://www.rero.ch/), [HES-SO](https://www.hes-so.ch/) / [HEG Genève](https://www.hesge.ch/heg/), [FH Graubünden](https://www.fhgr.ch/), [Università della Svizzera italiana](https://www.usi.ch/).
```
Just a space to be removed.

> ## SONAR's live services

Reviewer: ```suggestion
## ...to the services
```

> - **The [global portal](https://sonar.ch/):** this service aggregates all Swiss open access publications through content tracking. This mechanism harvests metadata from national and international repositories and creates a database. For each document with a full-text, a link to the source server (in principle the repository or the editor website) is available.

Reviewer: Afin que le lien soit plus visible et plus compréhensible

```suggestion
- **The global portal (on [sonar.ch](https://sonar.ch/)):** this service aggregates all Swiss open access publications through content tracking. This mechanism harvests metadata from national and international repositories and creates a database. For each document with a full-text, a link to the source server (in principle the repository or the editor website) is available.
```

> - **The dedicated portal:** For institutions that need it, SONAR can also act as a dedicated portal to host any type of digital library, not necessarliy open access. This type of portal benefits from an extended customer support and can be extensively customized by the client with its own visual identity as well as specific metadata fields. The [Haute école pédagogique du Valais](lien vers le portail dédié) uses this service and is SONAR's first client to go live.

Reviewer: Je renommerais ce service en **Institutional Repository as a Service**, en adaptant le texte en fonction et en précisant les deux modalités (shared, dedicated)

Je citerais les fonctionnalités principales:

> The main functionalities includes: publication deposit and validation, search, display, configuration, Switch edu-ID login. They have been publicly tested by various institutions throughout Switzerland and subsequently improved.
> The service operation and business model are well established and known by RERO, thanks to its RERO DOC experience. The service is completely new, among other things because it consists in a multi-tenant infrastructure in the cloud, in the four national languages and built on the latest version of Invenio.

Author: J'ai de la peine avec cette présentation. Je trouve que ça floute beaucoup la frontière entre le shared et le dedicated portal, alors que ce sont deux services vraiment très différents, à mon sens. Peut-être que c'est moi qui suis un peu lent mais je n'ai compris que le mois dernier la différence entre ces deux services (sans parler du portail global). J'essaierais de rendre très clair ce qui est proposé pour chaque service.

Le paragraphe qui décrit les fonctionnalités est très flou pour moi. La première phrase, on parle du dedicated, ou du shared ? Et la dernière phrase : le global et/ou le shared ?

Est-ce que le shared portal ne devrait pas être plus rattaché au global portal dans la logique, plutôt que de le présenter avec le dedicated alors que les deux services sont très différents ?

*capture d'écran du portail HEPVS*

A third service (the shared portal) is also available for institutions that need to host their open access publications. Less expensive and less customizable, this service offers a visibility of scholarly publications through the global portal and creates a dedicated collection for the client institution.

## The future

Two other institutions chose SONAR as the solution for their institutional archive, also preferring the dedicated option:
- [Università della Svizzera italiana](https://www.usi.ch/) 
- [HEP-BEJUNE](https://www.hep-bejune.ch/).

> 

Reviewer: Ajouter le remplacement progressif de RERO DOC. Bout de texte à intégrer, par exemple:

> SONAR is going to replace progressively RERO DOC in the course of 2021. The transition to production will be done gradually according to the planning of the interested institutions. Various discussions are under way.
